Comanche055/Comanche055_Index.md

```diff
@@ -0,0 +1,86 @@
+Source File                                                       Page Number
+
+CONTRACT_AND_APPROVALS.agc                                        #p.1
```

Comment:
Maybe add single spacing after the `#` and after the `.`, e.g. `# p. 1`, `# pp. 2-26`, ... for beauty and [consistency](http://blog.apastyle.org/apastyle/2015/03/when-and-how-to-include-page-numbers-in-apa-style-citations.html)? :bowtie:

Reply:
Roger. I didnt realize it was following APA style guidelines!

Reply:
Actually, its probably better for scripts to keep it simple: 1, 2-26, etc. Otherwise we put the responsibility on the users to parse out this unnecessary APA stuff. Ive updated the PR.

Reply:
Good call. Less is more 👍